compose/README.md

```diff
@@ -78,8 +78,10 @@ Run native macos sample:
 ## Multiplatform build
 
 ```console
-./compose/frameworks/support/jbdeps/android-sdk/downloadAndroidSdk
-export COMPOSE_CUSTOM_VERSION=1.1.0-beta04
+./frameworks/support/jbdeps/android-sdk/downloadAndroidSdk
+export ANDROID_SDK_ROOT=./frameworks/support/jbdeps/android-sdk/downloadAndroidSdk/darwin
```

Comment:
Thanks!

But calling downloadAndroidSdk should be enough. I will fix this soon, rewriting gradlew script

Jim also mention this issue recently.